### PR TITLE
updated Activity Overview read-only for MMIS

### LINKED ISSUE #4553: [Feature] Update Read-Only View for MMIS changes to Activity Overview
### Description and related issues
Update the Activity Overview section of the export view to display the new activity overview information, including the activity schedule information.

**Changes to Activities Overview:**
* Activity Start Date/End Date before the activity snapshot
* Add federal state split field
* Add activity snapshot field and comprehensive activity overview section (Short overview of activity language from HITECH is removed)

### Mockups or link to Figma
Data state - https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3630%3A20456
Empty state - https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3630%3A20456

### Acceptance criteria

- [ ] create stories for data and empty state
- [ ] update activity overview read-only component to display differently for HITECH and MMIS
- [ ] create MMIS version of read-only component to match the figma

### Testing criteria

| Given | When | Then | Covered |
| ----- | ---- | ---- | ------- |
|a HITECH APD| On the export view | Activity Overview section should be the same and not have any MMIS updates |         |
|a MMIS APD| On the export view (use Chromatic if page crashes) | Activity Overview section should match the data in the activity overview |         |
|a MMIS APD| Activity overview data is changed (added/deleted/edited) | Activity Overview section should match the changed data |         |
|a MMIS APD| Activity overview data is blank | Activity Overview section should match the empty state |         |

### PR DESCRIPTION
**Pull request by thetif**
Resolves #4553 

### Description
Updated the Activity read-only component. Pulled out the code for the Activity Overview into it's own component. Created components for HITECH Activity Overview and MMIS Activity Overview. The HITECH Activity Overview stays the same. The MMIS Activity Over was updated to match the Figma designs. Added tests and storybook for Activity Overview component.

I also added a knex template update for es6 style code.

Figma designs were updated:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4717%3A20126&t=df84z13A4Rpoo6s8-1
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4271%3A19818&t=df84z13A4Rpoo6s8-1

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| a HITECH APD with data | viewing Activity Overview | the information displays as expected |
| a HITECH APD without data | viewing Activity Overview | the information displays as expected |
| a MMIS APD with data | viewing Activity Overview | the information displays as expected |
| a MMIS APD with most of the data | viewing Activity Overview | the information displays as expected |
| a MMIS APD without data | viewing Activity Overview | the information displays as expected |

### Steps to manually verify this change

1. Open an HITECH APD
2. Click on Export and Submit
3. Click on Continue to Review
4. Scroll down to Activities and verify that it looks the same

You will not be able to view the read-only version of MMIS in the app, but you can verify it in [storybook](https://www.chromatic.com/build?appId=61d5b948cf6f17003a12bf77&number=1071).

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by tbolt — APPROVED** [review]

File: api/migrations-template.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * @returns { Promise<void> }
+ */
+export const up = async () => {};
+
+/**
+ * Make any changes that UNDO the up function side effects here (if possible)
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const down = async () => {};
```
Comment (amyd11) [inline api/migrations-template.js:13]: Thanks for throwing this in here

File: common/utils/constants.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ export const FUNDING_CATEGORY_TYPE = {
 
 export const FUNDING_CATEGORY_LABEL_MAPPING = {
   DDI: 'Design, Development, and Installation (DDI)',
-  MANDO: 'Maintenance & Operations (M&O)'
+  MANDO: 'Maintenance & Operations (M&O)',
+  MO: 'Maintenance & Operations (M&O)'
```
Comment (amyd11) [inline common/utils/constants.js:15]: Good add

**Review by amyd11 — APPROVED** [review]

**Comment by mirano-darren** [comment]
Looks good to me! I approved the changes in the PR
| Given | When | Then | Covered |
| ----- | ---- | ---- | ------- |
|a HITECH APD| On the export view | Activity Overview section should be the same and not have any MMIS updates | ✅ |
|a MMIS APD| On the export view (use Chromatic if page crashes) | Activity Overview section should match the data in the activity overview | ✅ |
|a MMIS APD| Activity overview data is changed (added/deleted/edited) | Activity Overview section should match the changed data | 🟨 Cannot test until export view is fully implemented |
|a MMIS APD| Activity overview data is blank | Activity Overview section should match the empty state | ✅ |

**Review by mirano-darren — APPROVED** [review]

**Review by jeromeleecms — APPROVED** [review]
Based on the PR (for HITECH) and the Storybook review (for MMIS) - this looks accurate.

**Review by SGilliamA1M — APPROVED** [review]
Changes look great!

**Pull request merged**